composer.lock

```diff
@@ -1969,7 +1969,7 @@
         },
         {
             "name": "symfony/dotenv",
-            "version": "v4.4.5",
+            "version": "v4.4.6",
```

Comment:
New release without changes.

Reply:
Thats how Symfony works, based on the subtree splitter

Reply:
Yep.